### PR TITLE
KEY-154: CLI content

### PR DESCRIPTION
**Pull request by jtbrolin**
- Adds "Copy command" button to homepage hero
- Adds CLI command and small content to /docs/quick-start page

### CODE REVIEW

File: docs/src/components/copy-command-button.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import { useState } from 'react';
+import { copyTextToClipboard } from '../utils';
+import Button from './button';
+
+export function CopyCommandButton() {
+  const copyCommandText = 'npm install @keystatic';
```
Comment (BarnabyBishop) [inline docs/src/components/copy-command-button.tsx:8]: This should be `npm create @keystatic`

Comment (BarnabyBishop) [reply docs/src/components/copy-command-button.tsx:8]: Actually make it `npm create @keystatic@latest`

Comment (jtbrolin) [reply docs/src/components/copy-command-button.tsx:8]: Ahh can't believe I typed that!

File: docs/src/components/copy-command-button.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      .then(() => {
+        setIsCopied(true);
+        setTimeout(() => {
+          setIsCopied(false);
+        }, 1500);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  };
```
Comment (BarnabyBishop) [inline docs/src/components/copy-command-button.tsx:22]: We should be able to convert this to async/await like so (I haven't tested this!). Bit easier on the eyes then promises.

```suggestion
  const handleCopyCommand = async () => {
    try {
      await copyTextToClipboard(copyCommandText);
      setIsCopied(true);
      setTimeout(() => {
        setIsCopied(false);
      }, 1500);
    }
    catch(err) {
      console.log(err);
    }
  };
```

File: docs/src/components/copy-command-button.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+  return (
+    <Button
+      className="border-keystatic-gray items-center py-0"
```
Comment (BarnabyBishop) [inline docs/src/components/copy-command-button.tsx:26]: On my screen the CLI command box looks a bit washed out and gets lost in the rest of the page:

CleanShot 2023-06-20 at 12 33 08@2x

Do you think we can make it stand out a bit more? Different background or darker border colour?

Basically I'm asking if you can make it pop. 😄

File: docs/src/components/copy-command-button.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      impact="light"
+      onClick={handleCopyCommand}
+      aria-label="Copy npm command for creating a Keystatic project"
+    >
+      <div className="flex gap-4 font-mono text-sm items-center leading-none">
+        {copyCommandText}
+
+        {isCopied ? (
```
Comment (BarnabyBishop) [inline docs/src/components/copy-command-button.tsx:34]: Love this detail

**Review by BarnabyBishop — COMMENTED** [review]
Looking good, just some small notes

File: docs/src/components/copy-command-button.tsx (hunk 3 above)
Comment (jtbrolin) [reply docs/src/components/copy-command-button.tsx:26]: Haha yeah I'll just make it the normal secondary Button style then and it'll pop more!

**Review by BarnabyBishop — APPROVED** [review]
💥

**Pull request merged**